From c67ed7fd2a243d06f60febc8ab41489b50b924b8 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 09:44:02 +0100
Subject: [PATCH 24/53] docs: update current maintainers of lksctp-tools

While at it, also credit all contributors that appear in the Git log.

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 AUTHORS | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/AUTHORS b/AUTHORS
index 98ffb31..01c25c0 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -1,36 +1,36 @@
-
 The initial developer and founder of the project:
 -------------------------------------------------
 La Monte H.P. Yarroll	<piggy@acm.org>
 
 Current Maintainer:
 -------------------
-Sridhar Samudrala	<sri@us.ibm.com>
+Vlad Yasevich		<vyasevic@redhat.com>
+Daniel Borkmann		<dborkman@redhat.com>
 
 Previous Maintainers:
 --------------------
+Sridhar Samudrala	<sri@us.ibm.com>
 Jon Grimm		<jgrimm@us.ibm.com>
 
 Contributors:
 -------------
 Karl Knutson		<karl@athena.chicago.il.us>
-
 Xingang Guo		<xingang.guo@intel.com>
-
 Daisy Chang		<daisyc@us.ibm.com>
-
 Inaky Perez-Gonzalez	<inaky.perez-gonzalez@intel.com>
-
 Hui Huang		<hui.huang@nokia.com>
-
 Dajiang Zhang		<dajiang.ahang@nolia.com>
-
 Ryan Layer		<rmlayer@us.ibm.com>
-
 Francois-Xavier Kowalski <francois-xavier.kowalski@hp.com>
-
-Vlad Yasevich            <vladislav.yasevich@hp.com>
-
-Frank Filz               <ffilz@us.ibm.com>
+Frank Filz		<ffilz@us.ibm.com>
+Ivan Skytte Jorgensen	<isj-sctp@i1.dk>
+Neil Horman		<nhorman@tuxdriver.com>
+Michael Biebl		<biebl@debian.org>
+Wei Yongjun		<yjwei@cn.fujitsu.com>
+Sam Miller		<samjmill@us.ibm.com>
+Andrei Pelinescu-Onciul	<andrei@iptel.org>
+Michele Baldessari	<michele@acksyn.org>
+Michael Tuexen		<tuexen@fh-muenster.de>
+Fan Du			<fan.du@windriver.com>
 
 Are you missing? drop us a line!
-- 
1.7.11.7

